resource/abstractions/auditing_rules.md

```diff
@@ -0,0 +1,474 @@
+---
```

Comment:
The Auditing rules documentation shouldn't be included in this PR. These updates are part of #52

Reply:
Correct, not sure exactly how this happened but it's been removed.